llama-index-integrations/readers/llama-index-readers-zyte-serp/README.md

```diff
@@ -0,0 +1,53 @@
+# LlamaIndex Readers Integration: Zyte-Serp
+
+ZyteSerp can be used to add organic search results from Google Search. It takes a `query` and returns top search results urls.
+
+## Instructions for ZyteSerpReader
+
+### Setup and Installation
+
+`pip install llama-index-readers-zyte-serp`
+
+1. **Install zyte-api Package**: Ensure the `zyte-api` package is installed to use the ZyteSerpReader. Install it via pip with the following command:
```

Comment:
I'm curious, can we declare a dependency on `zyte-api` package so that it gets installed automatically?

Reply:
Actually yes, we don't need this, my bad. We have already declared these dependencies.